k8s/ace-rp/kustomize/ace-rp/base/kustomization.yaml

```diff
@@ -33,5 +33,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: sandbox-ace-rp
-  newName: ghcr.io/trustbloc-cicd/sandbox-ace-rp
-  newTag: 0.1.9-snapshot-4c53fa6
+      newName: ghcr.io/trustbloc-cicd/sandbox-ace-rp
```

Comment:
Is this space intentional ? same comment for other cases

Reply:
no not intentional, it was by reformatter